charts/prometheus-node-exporter/templates/daemonset.yaml

```diff
@@ -41,7 +41,7 @@ spec:
       serviceAccountName: {{ include "prometheus-node-exporter.serviceAccountName" . }}
       containers:
         {{- $servicePort := ternary 8100 .Values.service.port .Values.kubeRBACProxy.enabled }}
-        - name: node-exporter
+        - name: {{ include "prometheus-node-exporter.name" . }}
```

Comment:
Imho this makes no sense. 
The vpa does'nt care for the container name.
It looks for " kind: DaemonSet".

Reply:
I still don't think a dynamic containername is a good idea.

Reply:
That's fair, I'll update the VPA object instead, to match the container name.